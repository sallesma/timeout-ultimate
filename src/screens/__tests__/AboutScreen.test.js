import React from 'react';
import { render } from '@testing-library/react-native';

import AboutScreen from '../AboutScreen';

describe('<AboutScreen />', () => {
  it('renders correctly', async () => {
    const { toJSON } = render(<AboutScreen />);

    expect(toJSON()).toMatchSnapshot();
  });
});
