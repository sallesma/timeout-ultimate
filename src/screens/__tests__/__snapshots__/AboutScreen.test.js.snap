// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutScreen /> renders correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    {
      "paddingBottom": 64,
    }
  }
  style={
    {
      "flex": 1,
      "padding": 16,
    }
  }
>
  <View>
    <Text
      style={
        {
          "fontSize": 20,
          "marginTop": 8,
        }
      }
    >
      About
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      Hello! I'm Martin Salles, the developer behind Timeout Ultimate. It's an application designed by ultimate players to help other players learn the rules of ultimate. It is and will remain free.
    </Text>
    <Text
      style={
        {
          "fontSize": 20,
          "marginTop": 8,
        }
      }
    >
      Acknowledgements
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      I would like to thank everyone who contributed to the development of Timeout Ultimate. Many of the questions come from Pablo's Disque Tu Sais game. Some questions were written by Pierre Gruau for the Monkeys. The application also uses the Hand Signals designed by Raphaël Candelier for the PUC and the WFDF. Finally, the English version was set up by Brandon Sneider justhuckitstudios+wfdfrulesquiz@gmail.com creator of UltiApp and https://GotSpirit.games .

The deployment of Timeout Ultimate on the stores is partly funded by Paris Université Club.
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      The rules of Ultimate are taken from the "WFDF Rules of Ultimate 2021-2024" produced by the World Flying Disc Federation.
    </Text>
    <Text
      style={
        {
          "fontSize": 20,
          "marginTop": 8,
        }
      }
    >
      Contribute
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      Whether it's adding, correcting questions or maintaining the application, any kind of help is welcome!
    </Text>
    <Text
      style={
        {
          "fontSize": 20,
          "marginTop": 8,
        }
      }
    >
      Feedback
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      The application is in the process of being created. I have a lot of ideas for the next step but I'd love to hear your opinion.
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
    >
      <Text
        style={
          {
            "color": "#2a9d9d",
            "fontSize": 16,
            "marginTop": 8,
          }
        }
      >
        Send a message
      </Text>
    </View>
    <Text
      style={
        {
          "fontSize": 20,
          "marginTop": 8,
        }
      }
    >
      Other projets
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
          "marginTop": 8,
        }
      }
    >
      In addition to this application, I am also working on the Disc In application, which allows you to prepare Ultimate practices.
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
    >
      <Text
        style={
          {
            "color": "#2a9d9d",
            "fontSize": 16,
            "marginTop": 8,
          }
        }
      >
        Disc In
      </Text>
    </View>
    <View
      style={
        {
          "marginTop": 32,
        }
      }
    >
      <Text
        style={
          {
            "color": "grey",
            "fontSize": 14,
          }
        }
      >
        Technical information
      </Text>
      <Text
        style={
          {
            "color": "grey",
            "fontSize": 14,
          }
        }
      >
        Timeout Ultimate
      </Text>
      <Text
        style={
          {
            "color": "grey",
            "fontSize": 14,
          }
        }
      >
        2021 - 
        2023
      </Text>
      <Text
        style={
          {
            "color": "grey",
            "fontSize": 14,
          }
        }
      >
        Version 
        , release-channel 
        mock
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
